src/product/ProductList.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  List,
+  Datagrid,
+  TextField,
+  ReferenceField,
+  NumberField,
+} from "react-admin";
+
+export const ProductList = () => (
+  <List resource="products" title="Posters">
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
+      <ReferenceField source="category_id" reference="categories">
+        <TextField source="id" />
+      </ReferenceField>
+      <NumberField source="width" />
+      <NumberField source="height" />
+      <NumberField source="price" />
+      <TextField source="thumbnail" />
+      <TextField source="image" />
```

Comment:
ImageField maybe?

Reply:
Fixed in https://github.com/slax57/admin-e-commerce-demo/pull/2 🙂